### PR TITLE
Delete meeting

### LINKED ISSUE #8: Delete Meeting
## Value Proposition

- As a dog owner
- I want to be able to delete already existing "meetings" 
- in order to let other people know that the meeting won't take place

## Description

Image

## Acceptance Criteria

- [x] MeetingDetail has button "löschen"
- [x] click on button "löschen" redirects to pages/index.js
- [x] after deleting a meeting, the components/MeetingEntry for the corresponding meeting deisappears from components/MeetingList in pages/index.js

## Tasks

- [x] create feature branch
- [x] implement "delete" button
- [x] write test to check if delte button is rendered

### PR DESCRIPTION
**Pull request by alexWaligorski**
fix #8

### CODE REVIEW

File: components/NewMeetingForm/NewMeetingForm.js [hunk 1]
```diff
@@ -143,6 +143,7 @@ export default function NewMeetingForm({
         alt="speichern Icon"
         source="/check-icon-round.svg"
         text="speichern"
+        functionality="safe"
```
Comment (PReichetanz) [inline components/NewMeetingForm/NewMeetingForm.js:146]: "safe" or "save"?

**Review by PReichetanz — APPROVED** [review]
Really nice work! Just one grammar issue ;)

File: components/NewMeetingForm/NewMeetingForm.js (hunk 1 above)
Comment (alexWaligorski) [reply components/NewMeetingForm/NewMeetingForm.js:146]: hehe.. teacher Paul and coach Paul are very right about this one. Will change it asap :D

Comment (LuYujin9) [reply components/NewMeetingForm/NewMeetingForm.js]: hehe...Coach Paul has eyes like eagle. Even the smallest things can't be hidden. 🤣

**Pull request merged**